casper/src/main/scala/coop/rchain/casper/ValidatorIdentity.scala

```diff
@@ -59,11 +56,6 @@ object ValidatorIdentity {
     )
   }
 
-  def getEnvVariablePassword[F[_]: Sync: EnvVars]: F[String] =
```

Comment:
Don't we want such kind of private key storage? I was thinking this code is actually used. I recall starting node with private key specified in env variable.

Reply:
Yes, we should create a proper solution, but until then half-built/leftover/idea/unused code should not be in the code base.